### PR TITLE
Added 4 German Courses

### PR DESCRIPTION
**Pull request by ecrax**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Add info | Improve Repo

### Description 
Added:

#### C++

* [C++ Grundlagen Tutorials von Pilzschaf](https://www.youtube.com/playlist?list=PLStQc0GqppuVs05kWvLBoHcWCULX3ueIM) - Pilzschaf

#### Java

* [Java Tutorial Deutsch - Programmieren lernen](https://www.youtube.com/playlist?list=PLgZuSc7xewde9zlJjmbLci0w9lV5BbCHE) - "Informatik - simpleclub"
* [Minecraft Plugins Programmieren für Anfänger](https://www.youtube.com/playlist?list=PLry1c-adUOIH3o2_K76jfznpw0-_3VpzY) - BiVieh

#### Unreal Engine

* [Unreal Engine 4 Tutorials (C++)](https://www.youtube.com/playlist?list=PLNmsVeXQZj7olLCliQ05e6hvEOl6sbBgv) - The Morpheus Tutorials

### Why is this valuable (or not)?
Because it expands the small collection of german programming courses.

### How do we know it's really free?
It's available on YouTube :D

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-de.md [hunk 1]
```diff
@@ -16,3 +30,8 @@
 * [Python-Kurs (Python 2)](https://www.python-kurs.eu/kurs.php)
 * [Python-Kurs (Python 3)](https://www.python-kurs.eu/python3_kurs.php)
 * [Python Tutorials Deutsch](https://www.youtube.com/playlist?list=PLNmsVeXQZj7q0ao69AIogD94oBgp3E9Zs)
+
+
+### Unreal Engine
+
+* [Unreal Engine 4 Tutorials (C++)](https://www.youtube.com/playlist?list=PLNmsVeXQZj7olLCliQ05e6hvEOl6sbBgv) - The Morpheus Tutorials
```
Comment (eshellman) [inline free-courses-de.md:37]: try to take the title from the resource, don't invent a title. Also, try to avoid picking a too-specific category.

Comment (ecrax) [reply free-courses-de.md:37]: Sorry, I have not seen that the title is incorrect.
Maybe the playlist would fit better under the c++ category?

File: free-courses-de.md [hunk 2]
```diff
@@ -1,17 +1,22 @@
 ### Index
 
 * [C++](#c++)
+* [Spieleentwicklung](#spieleentwicklung)
```
Comment (ecrax) [inline free-courses-de.md:4]: "Spieleentwicklung": Game Development in German

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**